spec/unit/homes_england/use_case/get_project_overview_spec.rb

```diff
@@ -0,0 +1,239 @@
+describe HomesEngland::UseCase::GetProjectOverview, focus: true do
```

Comment:
```suggestion
describe HomesEngland::UseCase::GetProjectOverview do
```
Also sidenote: you can use fdescribe instead or just :focus